CHANGELOG.MD

```diff
@@ -4,8 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project attempts to match the major and minor versions of [stactools](https://github.com/stac-utils/stactools) and increments the patch number as needed.
 
-## [v0.2.0]
+## [Unreleased]
+
+### Updated
+
+- Repo brought up to date with the current stactools-packages template contents as of 2022-02-18.
+- Edits to existing source and test code as part of the update were limited to:
+    - Format changes imparted by the `isort`, `yapf`, `flake8`, and `EditorConfig`
+    - A few minor changes to address `mypy` complaints in `utils.py` and `mtl_metadata.py`
+
+## [0.2.0] - 2021-07-20
 
 ### Changed
 
 - Upgrade to stactools 0.2.1.a2 (supporting PySTAC 1.0.0)
+
+[0.2.0]: <https://github.com/stactools-packages/landsat/releases/tag/v0.2.0>
```

Comment:
```suggestion
[Unreleased]: <https://github.com/stactools-packages/landsat/compare/v0.2.0..main>
[0.2.0]: <https://github.com/stactools-packages/landsat/releases/tag/v0.2.0>
```

Reply:
Added. Cool stuff. Was not aware of releases and comparison links.